.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "scroom"]
 	path = scroom
-	url = https://github.com/kees-jan/scroom.git
+	url = https://github.com/Group8Sep/scroom.git
```

Comment:
At the end of the project, this should point to the original repository. Accepting for now...